plugins/pluginmanagers/lazy.nix

```diff
@@ -1,169 +1,134 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
-let
-  cfg = config.plugins.lazy;
-  lazyPlugins = cfg.plugins;
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "lazy";
+  package = "lazy-nvim";
+  maintainers = [ ];
 
-  processPlugin =
-    plugin:
-    let
-      mkEntryFromDrv =
-        p:
-        if lib.isDerivation p then
-          {
-            name = "${lib.getName p}";
-            path = p;
-          }
-        else
-          {
-            name = "${lib.getName p.pkg}";
-            path = p.pkg;
-          };
-      processDependencies =
-        if plugin ? dependencies && plugin.dependencies != null then
-          builtins.concatMap processPlugin plugin.dependencies
-        else
-          [ ];
-    in
-    [ (mkEntryFromDrv plugin) ] ++ processDependencies;
+  hasSettings = false;
```

Comment:
We could signal our intent by flagging `hasSettings` with a FIXME or TODO.

Reply:
Already opened https://github.com/nix-community/nixvim/pull/3844 ;)